### PR TITLE
fix(all): Fix merge

### PR DESCRIPTION
**Pull request by Lordnoname**
Update .gitignore

fix(all): Fix last merge issues

### CODE REVIEW

**Comment by DevVersion** [comment]
Can you describe? Is this already tested?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+hs_err_pid*
+/bin/
+/target/
+.classpath
+/.settings/
+.project
+*.sla
+
+##### COSTUM #####
```
Comment (DevVersion) [inline .gitignore]: That's called `Custom`

File: src/main/java/de/thielpaul/vertretungsplan/gui/createobjects/DataManagerDialog.java [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+import de.thielpaul.vertretungsplan.core.interfaces.Teacher;
+import de.thielpaul.vertretungsplan.core.objects.School;
+import de.thielpaul.vertretungsplan.util.Pair;
+import de.thielpaul.vertretungsplan.util.Util;
+
+public class DataManagerDialog extends JFrame {
+
+	/**
+	 *
+	 */
```
Comment (DevVersion) [inline src/main/java/de/thielpaul/vertretungsplan/gui/createobjects/DataManagerDialog.java]: Please remove that unnecessary comments

File: src/main/java/de/thielpaul/vertretungsplan/gui/createobjects/NewSubjectDialog.java [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+import de.thielpaul.vertretungsplan.core.enums.Laboratory;
+import de.thielpaul.vertretungsplan.core.interfaces.Subject;
+import de.thielpaul.vertretungsplan.core.objects.School;
+
+public class NewSubjectDialog extends JFrame {
+
+	/**
+	 *
+	 */
```
Comment (DevVersion) [inline src/main/java/de/thielpaul/vertretungsplan/gui/createobjects/NewSubjectDialog.java]: As here the same

File: src/main/java/de/thielpaul/vertretungsplan/gui/substitution/SubstitutionDialog.java [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+import javax.swing.table.DefaultTableModel;
+
+import de.thielpaul.vertretungsplan.core.interfaces.Substitution;
+import de.thielpaul.vertretungsplan.manager.SubstitutionManager;
+
+public class SubstitutionDialog extends JFrame {
+
+	/**
+	 *
+	 */
```
Comment (DevVersion) [inline src/main/java/de/thielpaul/vertretungsplan/gui/substitution/SubstitutionDialog.java]: See upper comments

File: src/main/java/de/thielpaul/vertretungsplan/gui/timetable/ShowTimetablesDialog.java [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+import de.thielpaul.vertretungsplan.core.interfaces.timetable.Lesson.Time;
+import de.thielpaul.vertretungsplan.core.interfaces.timetable.Timetable;
+import de.thielpaul.vertretungsplan.core.objects.School;
+import de.thielpaul.vertretungsplan.manager.TimetableManager;
+
+public class ShowTimetablesDialog extends JFrame {
+
+	/**
+	 *
+	 */
```
Comment (DevVersion) [inline src/main/java/de/thielpaul/vertretungsplan/gui/timetable/ShowTimetablesDialog.java]: I just mark these for you, so you don't need to find them yourself

File: src/main/java/de/thielpaul/vertretungsplan/input/InstanceLoader.java [hunk 6]
```diff
@@ -86,4 +88,12 @@ public InstanceLoader() throws IOException {
...
 		return school;
 	}
-}
+
+	public void createNewFile(final File file) throws IOException {
+		file.createNewFile();
+		final FileWriter writer = new FileWriter(file, false);
+		writer.write("[]");
+		writer.close();
+		if(Application.DEBUG) System.out.println("Datei " + file.getName() + " wurde erstellt.");
```
Comment (DevVersion) [inline src/main/java/de/thielpaul/vertretungsplan/input/InstanceLoader.java]: Please space the Expression from the Statement

**Comment by Lordnoname** [comment]
@DevVersion I'm working on it. The comments were autogenerated :(

**Comment by DevVersion** [comment]
Okay, please fix and squash that, and then give me a quick summary of the changes (is this a working change? is this ready for merge?) If yes, you should add the `needs: merge` label.

Thanks in advance.
- CMO

**Comment by Lordnoname** [comment]
Fix typo in .gitignore
Remove unnecessary comments
Remove unused variables
Add space in InstanceLoader

Yes that changes fix the current issues caused by the last merge. I tested the most and important features.

**Comment by DevVersion** [comment]
Ouh shit....that are sick programming skills bro +1
I admire your skills...You are 1000 times better than me

**Pull request merged**